### PR TITLE
chore: 499 remove hours_of_day argument from analysis config and replace w…

### PR DESCRIPTION
**Pull request by MatthiasHauthDeltares**
…ith default value 24h

## Issue addressed
Solves #499 

## Code of conduct
- [x] I HAVE NOT added sensitive or compromised (test) data to the repository.
- [x] I HAVE NOT added vulnerabilities to the repository.
- [x] I HAVE discussed my solution with (other) members of the RA2CE team.

## What has been done?
Explain how you addressed the resolution of the related issue, what choices you made and why.

### Checklist
- [x] Code is formatted using our custom `black` and `isort` definitions.
- [x] Tests are either added or updated.
- [x] Branch is up to date with `master`.
- [x] Updated documentation if needed.

## Additional Notes (optional)
Add any additional notes or information that may be helpful.

### CODE REVIEW

**Comment by MatthiasHauthDeltares** [comment]
I still need to update the test data

File: ra2ce/analysis/analysis_config_data/analysis_config_data.py [hunk 1]
```diff
@@ -98,7 +98,6 @@ class AnalysisSectionLosses(AnalysisSectionBase):
     production_loss_per_capita_per_day: float = math.nan
     part_of_day: PartOfDayEnum = field(default_factory=lambda: PartOfDayEnum.DAY)
     performance: str = "diff_time"  # "diff_time" or "diff_length" relates to the used criticality metric
-    hours_per_day: float = 24
```
Comment (sahand-asgarpour) [inline ra2ce/analysis/analysis_config_data/analysis_config_data.py:101]: I suggest to keep hours_per_day but not use it. It allows us later to make use of it. For instance it is relevant if we have traffic data for 3 hours of the morning rush hour.

File: ra2ce/analysis/analysis_config_data/analysis_config_data_reader.py [hunk 2]
```diff
@@ -145,11 +145,6 @@ def _get_analysis_section_losses(self, section_name: str) -> AnalysisSectionLoss
             "duration_event",
             fallback=_section.duration_event,
         )
-        _section.hours_per_day = self._parser.getfloat(
-            section_name,
-            "hours_per_day",
-            fallback=_section.hours_per_day,
-        )
```
Comment (sahand-asgarpour) [inline ra2ce/analysis/analysis_config_data/analysis_config_data_reader.py:152]: I suggest to keep hours_per_day but not use it. Please see remark 1.

File: ra2ce/analysis/losses/losses_base.py [hunk 3]
```diff
@@ -97,10 +97,8 @@ def __init__(
         self.part_of_day: PartOfDayEnum = self.analysis.part_of_day
         self.analysis_type = self.analysis.analysis
         self.duration_event: float = self.analysis.duration_event
-        self.hours_per_day: float = self.analysis.hours_per_day
-        self.production_loss_per_capita_per_hour = (
-            self.analysis.production_loss_per_capita_per_day / self.hours_per_day
-        )
+        self.hours_per_day: float = 24  # 24 hours in a day
```
Comment (sahand-asgarpour) [inline ra2ce/analysis/losses/losses_base.py:100]:
```suggestion
        self.hours_per_day: float = self.analysis.hours_per_day  # 24 hours in a day as a default value
```
According to Remark 1.

**Review by sahand-asgarpour — CHANGES_REQUESTED** [review]
@MatthiasHauthDeltares Thanks for the changes. I placed a suggestion. Other than that it is good to go : )

File: ra2ce/analysis/analysis_config_data/analysis_config_data.py [hunk 4]
```diff
@@ -98,7 +98,6 @@ class AnalysisSectionLosses(AnalysisSectionBase):
     production_loss_per_capita_per_day: float = math.nan
     part_of_day: PartOfDayEnum = field(default_factory=lambda: PartOfDayEnum.DAY)
     performance: str = "diff_time"  # "diff_time" or "diff_length" relates to the used criticality metric
```
Comment (sahand-asgarpour) [inline ra2ce/analysis/analysis_config_data/analysis_config_data.py:100]:
```suggestion
    performance: str = "diff_time"  # "diff_time" or "diff_length" relates to the used criticality metric
    hours_per_day: float = 24
```

File: ra2ce/analysis/analysis_config_data/analysis_config_data_reader.py [hunk 5]
```diff
@@ -145,11 +145,6 @@ def _get_analysis_section_losses(self, section_name: str) -> AnalysisSectionLoss
             "duration_event",
             fallback=_section.duration_event,
         )
```
Comment (sahand-asgarpour) [inline ra2ce/analysis/analysis_config_data/analysis_config_data_reader.py:147]:
```suggestion
        )
        _section.hours_per_day = self._parser.getfloat(
        section_name,
        "hours_per_day",
        fallback=_section.hours_per_day,
        )
```

I suggest to keep hours_per_day but not use it. Please see remark 1.

File: ra2ce/analysis/analysis_config_data/analysis_config_data.py (hunk 4 above)
Comment (MatthiasHauthDeltares) [reply ra2ce/analysis/analysis_config_data/analysis_config_data.py:100]: I suggest the opposite to keep things as simple and unambiguous as possible. When we feel ready to have this feature and we figured out the math behind it, we will reintegrate it in the code.

Comment (MatthiasHauthDeltares) [reply ra2ce/analysis/analysis_config_data/analysis_config_data.py:100]: Plus, we still need to use it with the default value of 24h.

File: ra2ce/analysis/losses/losses_base.py (hunk 3 above)
Comment (MatthiasHauthDeltares) [reply ra2ce/analysis/losses/losses_base.py]: Agreed

**Review by sahand-asgarpour — APPROVED** [review]

**Pull request merged**